### DISCUSSION TITLE
TypeScript tsconfig.json defaults?

### DISCUSSION THREAD

**Discussion by ablakey** [parcel-bundler/parcel#8814] [Q&A]
If I do NOT include a `tsconfig.json`, I assume Parcel assumes a default config.

If I want to make a singular change, such as enabling `strictPropertyInitialization`, is there a way to do that without completely clobbering Parcel's defaults (because I don't want to personally manage all the settings, I just want to add one)?

**Comment by mischnic** [comment]
There's only a handful of properties that Parcel uses from tsconfig:
https://github.com/parcel-bundler/parcel/blob/1fd82417122fccd5ea465b2c237f736fc3d2d53d/packages/transformers/js/src/JSTransformer.js#L198-L238
And most of them you need to set for tsc typechecking to work anyway (depending on the project).
So there aren't really any defaults
